### ISSUE TITLE
where did 'Location' Header go?

### ISSUE THREAD

**Issue by cubap** [CenterForDigitalHumanities/rerum_server#126]
It is in the console, but not apparently reachable in the JS...
Maybe

https://stackoverflow.com/a/56959114 it is not able to survive a CORS transfer without being called out explicitly?

**Comment by thehabes** [comment]
oof https://developers.google.com/web/updates/2015/03/introduction-to-fetch#response_types

**Comment by thehabes** [comment]
oof https://stackoverflow.com/questions/43344819/reading-response-headers-with-fetch-api.  It says just read the body.  I don't like this.  The internet is broken.

**Comment by thehabes** [comment]
This restriction for CORS respects the values in `access-control-expose-headers` and `access-control-allow-headers`.  So we would have to update the responses out of rerum_server with these headers the let JS processors access the `Location` header if they aren't already doing it.  If they are already doing it, then IDK.

**Comment by thehabes** [comment]
our rerum_server only does 

```java
response.addHeader("Access-Control-Allow-Headers", "Content-Type");
response.addHeader("Access-Control-Allow-Methods", "GET,OPTIONS,HEAD,PUT,PATCH,DELETE,POST");
```

So we need to update that line in `processRequestyBody()`.

**Comment by cubap** [comment]
Not urgent. Just noting it, since it was tangential to what I was working on.

**Comment by thehabes** [comment]
Agreed.  Good catch though, that slipped right passed me.  Put it on the pile.
